contrib/python-resolvelib/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-resolvelib"
+pkgver = "1.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+checkdepends = ["python-packaging", "python-pytest"]
+depends = ["python"]
+pkgdesc = "Resolve abstract dependencies into concrete ones"
+maintainer = "Mara <177581589+catgirlconspiracy@users.noreply.github.com>"
+license = "ISC"
+url = "https://github.com/sarugaku/resolvelib"
+source = f"$(PYPI_SITE)/r/resolvelib/resolvelib-{pkgver}.tar.gz"
```

Comment:
make this `{url}/archive/refs/tags/{pkgver}.tar.gz` and then you can drop the patch above (if pypi tarballs are scuffed and miss data feel free to use the github tarballs)

Reply:
...so apparently the Github tarball contains even more tests that were missing from the pypi package, which require at least one other external module; do I just package that one as well? (no transitive deps)

Reply:
go for it